dcpy/lifecycle/ingest/run.py

```diff
@@ -35,32 +35,21 @@ def run(
         config.file_format, file_path, dir=staging_dir, output_filename=init_parquet
     )
 
-    post_parquet = "post.parquet"
     transform.run_processing_steps(
         config.name,
         config.processing_steps,
         staging_dir / init_parquet,
-        staging_dir / post_parquet,
+        staging_dir / config.filename,
     )
 
-    ## logic to apply transformations based on parsed config/template. Something like this
-    # for step in config.processing.steps:
-    #    func = getattr(processing, step)
-    #    func(local_parquet_path)
-    ##
-
-    # if not skip_archival:
-    #    recipes.archive(
-    #        import_config, PARQUET_PATH
-    #    )
-
-    raise NotImplementedError()
+    if not skip_archival:
+        recipes.archive_dataset(config, staging_dir / config.filename)
 
 
 app = typer.Typer(add_completion=False)
 
 
-@app.command("run")
+@app.command()
 def _cli_wrapper_run(
```

Comment:
Since we changed the structure from where the CLI command is triggered, perhaps just move the code below into `dcpy/lifecycle/_cli.py` for simplicity?

Reply:
Maybe. This is following Alex's structure for his (to some extent - I obviously tweaked it a bit since there's only one ingest command for now). But I think the idea is that cli file really just has the links to the sub-apps/commands and not logic specific to those commands.

Definitely something that's open for discussion, but I'd like to keep the same convention for now

Reply:
I think I might lean with what you're suggesting though. Warrants a broader discussion!

Reply:
Yeah, could see either way. Open to chat, but maybe we just let things evolve, and make a decision when we know better what we need

Reply:
Makes sense why you changed it, @fvankrieken . Agree, would be nice to have the same convention across the board.